source/ConditionSet.cpp

```diff
@@ -92,6 +93,29 @@ bool ConditionSet::IsEmpty() const
 
 
 
+// Do text replacement throughout this ConditionSet.
+ConditionSet ConditionSet::Substitute(const map<string, string> &subs) const
+{
+	ConditionSet result = *this;
+	DataNode someNode = DataNode();
+	for(Expression &expression : result.expressions)
+	{
+		expression.name = Format::Replace(expression.name, subs);
```

Comment:
Dang, I didnt paste that to the branch? My local file has that include. :-/
Fixing it right away.